<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_Dialog</class>
 <widget class="QDialog" name="Login_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>329</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>496</width>
    <height>329</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>496</width>
    <height>329</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/bq/000.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>　登录　</string>
    </attribute>
    <widget class="QLineEdit" name="line_password">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="number">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>72</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_number">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_login_login">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>登陆</string>
     </property>
    </widget>
    <widget class="QLabel" name="password">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>72</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_login_clear">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string> 注册　</string>
    </attribute>
    <widget class="QLabel" name="register_number">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>72</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="register_line_number">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="register_password">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>72</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="register_line_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="register_confirmpassword">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>72</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="register_confirm_line_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="register_confirm">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="register_clear">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>line_number</tabstop>
  <tabstop>line_password</tabstop>
  <tabstop>button_login_clear</tabstop>
  <tabstop>button_login_login</tabstop>
  <tabstop>register_line_number</tabstop>
  <tabstop>register_line_password</tabstop>
  <tabstop>register_confirm_line_password</tabstop>
  <tabstop>register_confirm</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
